docs/management/managing-saved-objects.asciidoc

```diff
@@ -39,6 +39,17 @@ have multiple environments for development and production.
 Import and export also work well when you have a large number
 of objects to update and want to batch the process.
 
+[float]
+==== Compatibility across versions and downgrades
+
+We do not guarantee compatibility of a Saved Object exported from newer Kibana version and imported to older Kibana version:
```

Comment:
@rudolf I thought importing objects with a higher `migrationVersion` to an older instance would just throw an error for the object, but I can't find this behavior in `src/core/server/saved_objects/import/import_saved_objects.ts`?

Reply:
Based on this forum post https://discuss.elastic.co/t/copy-visualisation-and-dashboards/225958 I don't think it's producing a nice error message, just failing with a generic "Could not be processed"

Reply:
Any update on this?

Reply:
Before creating any objects we migrate them, so trying to import a newer object will fail the migration and therefore the `SavedObjectsRepository#bulkCreate`.

The API correctly responds with:
```
{"statusCode":422,"error":"Unprocessable Entity","message":"Document \"d3d7af60-4c81-11e8-b3d7-01146121b73d\" has property \"index-pattern\" which belongs to a more recent version of Kibana [8.1.0]. The last known version is [7.6.0]"
```

But we like @flash1293 said we don't display this to users, I'll open an issue for this.

Reply:
When importing using the deprecated `.json` file format we correctly surface the error in the UI.

Reply:
Created https://github.com/elastic/kibana/issues/66387